docs/mint.json

```diff
@@ -119,7 +119,8 @@
         {
           "group": "Java",
           "pages": [
-            "instrumentations/java/ebpf"
+            "instrumentations/java/ebpf",
+            "instrumentations/java/java"
```

Comment:
perhaps make the native java page first, as this is relevant to most of our users